src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const fullPrice = days * 40;
+
+  if (days < 3) {
+    return fullPrice;
+  } else if (days < 7) {
+    return fullPrice - 20;
+  } else {
+    return fullPrice - 50;
+  }
```

Comment:
Last else is redundant here
```suggestion
  return fullPrice - 50;
```